src/components/camera-tool.js

```diff
@@ -156,6 +156,16 @@ AFRAME.registerComponent("camera-tool", {
         renderer.readRenderTargetPixels(this.renderTarget, 0, 0, width, height, this.snapPixels);
         pixelsToPNG(this.snapPixels, width, height).then(file => {
           const { entity, orientation } = addMedia(file, "#interactable-media", undefined, true);
+          entity.addEventListener(
+            "media_resolved",
+            () => {
+              window.APP.hubChannel.sendMessage(
+                { contentType: "image/jpg", src: entity.components["media-loader"].data.src },
```

Comment:
it's not, jus guessing we will have other spawn types soon. That said I should probably get the content type that actually got resolved since I think this is wrong. Will just strip out for now.